scripts/addr2line.py

```diff
@@ -347,12 +363,17 @@ def __init__(
         )  # fail fast if there is something wrong with the exe resolver
         self.parser = self.BacktraceParser()
 
+    def debug(self, *args: Any):
+        if self._debug:
+            print('DEBUG >>', *args, file=sys.stderr)
```

Comment:
Why not using a logger?

Reply:
is it because we want to print to stderr?

Reply:
otherwise, LGTM

Reply:
I do want to print it to stderr, yes, to separate it from the primary output which goes to stdout (and which would often be redirected elsewhere). I don't think that excludes using a logger though, does it?

I didn't use a logger because this was easier I think.

Reply:
@fruch WDYT?  Should we use a logger in seastar-addr2line or printing to stderr good enough?

Reply:
Logger is needed if you want to have context information, like timestamps, filename, line numbers and such.

Or when you have multiple destinations, like stdout and log file.

For one file script like this one its not a must.

Reply:
Great, I elect not to use a logger then (though ofc still happy to change this if the wind should shift).